### PR TITLE
Add P0 Components.

### PR DESCRIPTION
**Pull request by dhruvin25799**
Added P0 Components and Live Website files.
Please review
Check components.css for CSS file for components.
main.css and example_main.css are CSS files for website.

### CODE REVIEW

**Comment by ritikaawasthy** [comment]
I think alerts should have a dismiss button otherwise looks great

File: components.css [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+    padding: 0.5rem;
+    margin: 0.5rem;
+}
+
+.alert p{
+    margin: 0;
+}
+
+.alert.error{
+    background-color: #dc3545;
```
Comment (anshulraheja) [inline components.css]: suggestion to use variables for colors

File: components.css [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+.alert.info{
+    background-color: #17a2b8;
+    color: white;
+}
+
+/* Avatar */
+.avatar{
+    border-radius: 50%;
+    width: 3rem;
+    height: auto;
```
Comment (anshulraheja) [inline components.css]: I don't think height auto will have any effect. Width will manage the size

File: components.css [hunk 3]
```diff
@@ -0,0 +1,408 @@
...
+    position: absolute;
+    border-radius: 50%;
+    width: 1rem;
+    height: 1rem;
+    bottom: 0;
+    right: 0;
+    border: 2px solid white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (anshulraheja) [inline components.css]: No need of display flex property

File: components.css [hunk 4]
```diff
@@ -0,0 +1,408 @@
...
+  text-decoration: none;
+  border: none;
+  border-radius: 0.5rem;
+  transition: all 0.3s ease 0s;
+}
+
+.btn.btn-primary {
+  color: white;
+  background: #dc0000;
+  cursor: pointer;
```
Comment (anshulraheja) [inline components.css]: give cursor pointer in btn class and remove from all secondary classes

File: components.css [hunk 5]
```diff
@@ -0,0 +1,408 @@
...
+
+.input-control label {
+position: absolute;
+top: 0;
+right: 1px;
+bottom: 1px;
+left: .5em;
+z-index: 1;
+height: 1em;
+font-size: 13px;
```
Comment (anshulraheja) [inline components.css]: use rem

**Review by anshulraheja — APPROVED** [review]

**Pull request merged**